### PR TITLE
Add documentation about upcoming change to logout

### PR DESCRIPTION
**Pull request by orenyk**
To be merged after comms go out.

### CODE REVIEW

File: _pages/oidc.md [hunk 1]
```diff
@@ -424,11 +443,13 @@ Here's an example logout request:
...
+  client_id=${CLIENT_ID}&
   post_logout_redirect_uri=${REDIRECT_URI}&
   state=abcdefghijklmnopabcdefghijklmnop
 ```
 
+Note that, as per the spec, Login.gov will display a Logout confirmation screen to users on logout; users will need to click a button to complete the logout process. This protects against forged logout request attacks.
```
Comment (pauldoomgov) [inline _pages/oidc.md:451]: Is it worth adding something here or in "Logout response" that also notes that nothing is returned if the user does not click the Logout Confirmation?  (Thinking of some repeat questions from the team on this that partners will also probably have.)

File: _pages/oidc.md [hunk 2]
```diff
@@ -405,17 +407,34 @@ This public key is rotated periodically (on at least an annual basis). It is imp
...
 Login.gov does not support Single Logout (SLO). The logout action will terminate the user's session at Login.gov but will not end any other potentially active sessions within service provider applications. For example, if a user signs in to applications A and B through Login.gov, a logout request from A will end their Login.gov session, but will not affect the session in application B.
 
+### IMPORTANT: Changes to Logout
+
+Login.gov will begin accepting the `client_id` parameter in logout requests to identify the integration sending the logout requests in addition to the `id_token_hint` parameter in the coming weeks. Following that, **Login.gov will stop supporting the `id_token_hint` parameter in logout requests** - requests that include that parameter will display an error page to the user.
+
+The rollout dates are as follows:
+
+* **Thursday, September 21, 2022** - `client_id` accepted in logout requests in the **sandbox** environment
+* **Monday, September 24, 2022** - `client_id` accepted in logout requests in the **production** environment
```
Comment (pauldoomgov) [inline _pages/oidc.md:423]: Thursday, September 29th is the current proposed date for Logout Confirmation + client_id on logout.

Comment (orenyk) [reply _pages/oidc.md:423]: thanks! I also realized 9/21 is a Wednesday 🤦

File: _pages/oidc.md (hunk 1 above)
Comment (orenyk) [reply _pages/oidc.md:451]: Added some more explicit language, thanks!

**Review by pauldoomgov — APPROVED** [review]
LGTM!

**Pull request merged**